docs/resources/telemetry_information.md

```diff
@@ -0,0 +1,77 @@
+# OpenVINO™ Telemetry {#openvino_docs_telemetry_information}
+
+@sphinxdirective
+
+To facilitate debugging and further development, OpenVINO™ asks its users for 
+a permission to collect telemetry data. It will not be collected 
+without an explicit consent on your part and will cover only OpenVINO™ usage information.
+It does not extend to any other Intel software, hardware, website usage, or other products. 
+
+Google Analytics is used for telemetry purposes. The data is collected anonymously 
+and does not allow identifying users it comes from. 
+
+Enable or disable Telemetry reporting
+======================================
+
+First-run consent
+--------------------------------------
+
+On the first run of an application that collects telemetry data, you will be prompted 
+to opt in or out of telemetry collection with the following telemetry message: 
+
+.. code-block:: console
+
+   Intel would like your permission to collect software performance and usage data
+   for the purpose of improving Intel products and services. This data will be collected
+   directly by Intel or through the use of Google Analytics. This data will be stored 
+   in countries where Intel or Google operate.
+
+   You can opt-out at any time in the future by running 'opt_in_out --opt_in'.
+   
+   More Information is available at docs.openvino.ai.
+
+   Please type 'Y' to give your consent or 'N' to decline.
+
+Choose your preference by typing 'Y' to enable or 'N' to disable telemetry. Your choice will 
+be confirmed by a corresponding disclaimer. If you do not reply to the telemetry message, 
+your telemetry data will not be collected. 
+
+For the Neural Network Compression Framework (NNCF), which is not a command line application, 
+the telemetry message will not display. Telemetry data will only be collected from NNCF 
+if you have explicitly provided consent in another OpenVINO tool.
+
+
+Changing consent decision
+--------------------------------------
+
+You can change your data collection decision with the following command lines: 
+
+`opt_in_out --opt_in` - enable telemetry
+
+`opt_in_out --opt_out` - disable telemetry
+
+
+Telemetry Data Collection Details
+======================================
+
+.. tab:: Telemetry Data Collected 
+
+   * Failure reports 
+   * Error reports 
+   * Usage data 
+
+.. tab:: Tools Collecting Data
+
+   * Model Optimizer 
+   * Model Downloader 
```

Comment:
Model Converter and Model Quantizer also collect data (they show the consent message). I don't know if there are other tools missing from this list; it would be good to check.

Reply:
I asked about it but haven't received any confirmation. I will just include the two and we'll polish the list if necessary after the initial publishing. Thanks for the catch.